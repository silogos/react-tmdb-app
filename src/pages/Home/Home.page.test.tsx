import { describe, it, expect } from "vitest";
import HomePage from "./Home.page";
import { render } from "@testing-library/react";

describe("Home Page", () => {
  it("renders correctly", () => {
    const component = render(<HomePage />);
    expect(component).toBeTruthy();
  });

  it("renders toMatchSnapshot", () => {
    const component = render(<HomePage />);
    expect(component).toMatchSnapshot();
  });
});
