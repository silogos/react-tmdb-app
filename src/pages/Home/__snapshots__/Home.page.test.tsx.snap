// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Home Page > renders correctly 1`] = `
<div>
  <div
    data-testid="MovieList"
  />
</div>
`;
